instrumentation/faraday/lib/opentelemetry/instrumentation/faraday/middlewares/tracer_middleware.rb

```diff
@@ -42,14 +44,10 @@ def call(env)
           attr_reader :app
 
           def span_creation_attributes(http_method:, url:)
-            instrumentation_attrs = {
-              'http.method' => http_method, 'http.url' => url.to_s
-            }
             config = Faraday::Instrumentation.instance.config
-            instrumentation_attrs['peer.service'] = config[:peer_service] if config[:peer_service]
-            instrumentation_attrs.merge(
-              OpenTelemetry::Common::HTTP::ClientContext.attributes
-            )
+            attributes = OpenTelemetry::Common::HTTP::RequestAttributes.from_request(http_method, url, config)
+            attributes['peer.service'] = config[:peer_service] if config[:peer_service]
```

Comment:
same as https://github.com/open-telemetry/opentelemetry-ruby/pull/769/files#r638374660

Reply:
Oh, hm this is how it was before. disregard i guess?